game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/RollValue.java

```diff
@@ -0,0 +1,31 @@
+package games.strategy.triplea.delegate.power.calculator;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Value;
+
+/**
+ * Holds the value of a dice roll.
+ *
+ * <p>Handles infinite values and correctly limits the value
```

Comment:
More explanation about the expected behavior here would help. It's not clear what is meant by an infinite roll value, and what 'correctly limiting a value' would mean. EG: what is correct? What would be incorrect? What does it mean to 'limit' a roll value?